scipy/stats/_morestats.py

```diff
@@ -2710,6 +2710,11 @@ def levene(*samples, center='median', proportiontocut=0.05):
     pvalue : float
         The p-value for the test.
 
+    See Also
+    --------
+    fligner : A non-parametric test for the equality of k variances
+    bartlett : A parametric test for equality of k variances in normal samples
```

Comment:
```suggestion
    fligner: A non-parametric test for the equality of k variances
    bartlett: A parametric test for equality of k variances in normal samples
```

Reply:
Items in `See also` need the space. It's kind of a directive like the parameters.

Reply:
@j-bowhay, the [NumPy docstring standard](https://numpydoc.readthedocs.io/en/v1.5.0/format.html) requires those spaces.  It is a convention that is used throughout the various sections of the docstring standard.

Reply:
Ah thanks, I quickly skimmed through the file  and saw `kstatvar` didnt have the spaces so I wrongly assumed that was the standard!

Reply:
Indeed, I will fix this everywhere in this file here if fine for everyone.